ibm/service/vpc/resource_ibm_is_lb_pool_member_test.go

```diff
@@ -70,7 +70,8 @@ func TestAccIBMISLBPoolMember_basic_network(t *testing.T) {
 	nlbName := fmt.Sprintf("tfnlbcreate%d", acctest.RandIntRange(10, 100))
 	nlbName1 := fmt.Sprintf("tfnlbupdate%d", acctest.RandIntRange(10, 100))
 
-	sshname := "terraform-test-ssh-key"
+	// sshname := "terraform-test-ssh-key"
+	sshname := "ssh-test-bhavesh"
```

Comment:
generate a new suffix to 'ssh-key' instead of using name. refer other test files for it

Reply:
yeah, I will revert this is to existing.